README.md

```diff
@@ -26,6 +26,7 @@ The compare tool allows you to use a local config file (`src/config/local.js`) t
 Here, you can customize the following options:
 
 - `mapboxGlAccessToken`: Your Mapbox GL token to allow style reads
+  - `mapboxBaseApiUrl`: (optional) Support for a Mapbox flavored style that is served from a different server other than `api.mapbox.com`, e.g., Mapbox Atlas.
```

Comment:
Super minor thing I just noticed: We don't want this indented since it's its own option and not a sub-option!

Reply:
Great!  No worries, fixed.